_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,49 @@
+Something old, something new...
+
+
+Even though it’s only June, our HackIllinois Summer team is already well under way in planning the event for next February! We’ve been exploring tons of new possibilities for what the future of our yearly gathering will look like, and fine tuning the things that didn’t go as well last year.
```

Comment:
not sure if we want to end paragraph with "things that didn't go as well last year" or even mention if we aren't going to elaborate -- asking for feedback later on is forum enough for people to give their opinion if they truly felt something did go wrong